src/main/java/org/betonquest/betonquest/modules/logger/custom/debug/config/SimpleDebugConfig.java

```diff
@@ -0,0 +1,99 @@
+package org.betonquest.betonquest.modules.logger.custom.debug.config;
+
+
+import org.betonquest.betonquest.api.config.ConfigurationFile;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * This is a simple debug configuration.
+ */
+public class SimpleDebugConfig implements DebugConfig {
```

Comment:
This is premature abstraction: The only use case is to be further extended, so this class and `FileDebugConfig` could be collapsed into one class. Abstraction should happen at a later point when it is know what changes and what stays the same between different implementations (but there is just one right now).